openwisp_radius/api/freeradius_views.py

```diff
@@ -97,16 +97,20 @@ def _check_client_ip_and_return(self, request, uuid):
         ip_list = self._get_ip_list(uuid)
 
         for ip in ip_list:
-            try:
-                if ipaddress.ip_address(client_ip) in ipaddress.ip_network(ip):
-                    return (AnonymousUser(), uuid)
-            except ValueError:
-                invalid_addr_message = _(
-                    'Request rejected: ({ip}) in organization settings or '
-                    'settings.py is not a valid IP address. '
-                    'Please contact administrator.'
-                ).format(ip=ip)
-                raise AuthenticationFailed(invalid_addr_message)
+            if '/' in ip:  # Check for CIDR notation (IP range)
```

Comment:
I am not a fan of this approach.

See the following:

```
In [2]: ipaddress.ip_network('10.0.0.1')
Out[2]: IPv4Network('10.0.0.1/32')

In [3]: ipaddress.ip_network('fe80::714:f09:20f3:d39d')
Out[3]: IPv6Network('fe80::714:f09:20f3:d39d/128')
```

As you can see, initialitizing ip_network() with a single IP is possible and yields a /32 on ipv4 or a /128 on ipv6.

So:

```
In [4]: ipaddress.ip_address('10.0.0.1') in ipaddress.ip_network('10.0.0.1')
Out[4]: True
```

Would work. So I am not sure what problem your this is really solving. To me it looks like this code isn't implementing anything which the current code doesn't already do.